src/api.js

```diff
@@ -27,4 +27,16 @@ export const api = {
       }
     },
   },
+  getCommentsByArticleId: {
+    get: async (id) => {
+      try {
+        const response = await axios.get(`${baseUrl}/api/articles/${id}/comments`);
+
+        return response.data?.comments || [];
```

Comment:
depending on how your backend is setup, we should be getting a happy path response even if there are no comments. So in the component we could just handle there being no comments by conditionally displaying a message in that case. So we might not need this bit of logic?